README.md

```diff
@@ -30,3 +30,10 @@ NEXT_PUBLIC_BASE_URL=http://localhost:3000
 ### Launch the app
 
 Then install dependencies using `npm install`, run the dev server using `npm run dev`
+
+### Generate seeds
+
+If you want to generate random seed data, you can use the hasura console to export the Skill table to JSON
```

Comment:
Wouldn't the Skill table be empty when I'm getting started?

Reply:
There is a list of skills taken from the old form that is inserted as a migration